casr/src/bin/casr-msan.rs

```diff
@@ -0,0 +1,290 @@
+use casr::util;
```

Comment:
The file looks likes mostly a copy-paste of `casr-san.rs`. We may just add `MemorySanitizer` support to `casr-san`, as it does not state any exact sanitizer in its name.